blocks/carousel/carousel.css

```diff
@@ -0,0 +1,216 @@
+main .carousel-container {
+    padding: 0;
+}
+
+main .carousel {
+    display: flex;
+    scroll-snap-type: x mandatory;
+    overflow-x: scroll;
+    scroll-behavior: smooth;
+    position: relative;
+    height: 650px;
+}
+
+main .carousel picture {
+    position: absolute;
+    z-index: -1;
+    top: 0;
+    left: 0;
+    bottom: 0;
+    right: 0;
+    box-sizing: border-box;
+}
+
+main .carousel img {
+    object-fit: cover;
+    width: 100%;
+    height: 100%;
+    object-position: 90%;
+}
+
+main .carousel > div {
+    width: 100%;
+    flex-shrink: 0;
+    scroll-snap-align: start;
+    position: absolute;
+    top: 0;
+    left: 0;
+    height: 650px;
+    align-items: flex-end;
+    display: flex;
+    background-image: linear-gradient(#0000, #0008);
+    color: white;
+    opacity: 0;
+    transition: opacity 0.3s ease-in-out 0.3s;
+}
+
+main .carousel > div.selected {
+    opacity: 1;
+}
+
+main .carousel div.carousel-text {
+    text-align: center;
+    width: 500px;
+    margin: 0 auto;
+}
+
+
+main .carousel::-webkit-scrollbar {
+    display: none;
+}
+
+main .carousel-wrapper {
+    max-width: unset;
+}
+
+main .carousel-wrapper .carousel-buttons {
+    text-align: center;
+    margin-top: -40px;
+    position: absolute;
+    display: flex;
+    right: 0;
+    left: 0;
+    justify-content: center;
+}
+
+main .carousel-buttons button {
+    top: 0;
+    display: block;
+    height: 15px;
+    width: 15px;
+    padding: 0 !important;
```

Comment:
don't use !important

Reply:
I have added !important here since we already have padding: 15px 75px !important; under a.button:any-link, button CSS and due to which that padding was impacting buttons shape. Hence in order to neutralise that effect, I need to use here ! important.